examples/read_module_protocol.cf

```diff
@@ -0,0 +1,44 @@
+bundle agent cache_maintenance
+# Creates a module protocol cache, refreshes it if minute is 30-35
+{
+    vars:
+      "file"
+        string => "$(this.promise_dirname)/cached_module";
+
+  classes:
+    "cache_refresh"
+      if => not(fileexists("$(file)"));
```

Comment:
you shouldn't have to quote $(file) here.

Reply:
I disagree - my style more clearly shows that it expands to a string.

Reply:
I also prefer the quotes.

Reply:
that's an interesting difference between traditional developers and cfengine users, various users have been pushing to kill quotes where un-necessary